tests/unit/utils/test_helpers.py

```diff
@@ -0,0 +1,106 @@
+import pytest
+from cellengine.utils.helpers import alter_keys, to_camel_case
+
+
+def test_alter_keys_converts_dict_to_camel_case():
+    d = {
+        "_id": "foo",
+        "experiment_id": "bar",
+        "fcsFile": "baz",
+    }
+    r = alter_keys(d, to_camel_case)
+    assert {"_id", "experimentId", "fcsFile"} <= r.keys()
```

Comment:
Here and in the other tests, probably better to use `==` to make sure the result doesn't also contain `experiment_id`, right?